transforms/unchain-variables.js

```diff
@@ -9,6 +9,9 @@ module.exports = function(file, api, options) {
     .find(jscodeshift.VariableDeclaration)
     .filter(variableDeclaration => (
       variableDeclaration.value.declarations.length > 1
```

Comment:
Thanks for the feedback! I actually prefer it as-is, but I don't feel strongly either way. Feel free to modify.